src/k8s/cmd/k8s/k8s_kubectl.go

```diff
@@ -24,15 +24,15 @@ func newKubectlCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
 				return
 			}
 
-			if status, initialized, err := client.NodeStatus(cmd.Context()); err != nil {
+			if resp, initialized, err := client.NodeStatus(cmd.Context()); err != nil {
```

Comment:
I see `resp` and `response` mixed in the `cmd`, should we unify those to be one of the 2?